libs/deal-ticket/src/components/market-selector.tsx

```diff
@@ -273,9 +273,8 @@ export const MarketSelector = ({ market, setMarket, ItemRenderer }: Props) => {
     <>
       {!dialogContent && selectorContent}
       <Dialog
-        titleClassNames="uppercase font-alpha"
-        contentClassNames="left-[0px] top-[99px] h-[calc(100%-99px)] border-0 translate-x-[0] translate-y-[0] border-none overflow-y-auto"
```

Comment:
these styles are crucial for display the market selector on mobile view. In my opinion they must stay

Reply:
we were thinking of setting 2 or 3 fixed sizes on the dialog and to not override the content class names anymore. if we add a set size for mobile dialog, do you think it would do the trick?

Reply:
I need to set top and height of the dialog there. Sizes can't provide this

Reply:
I see `large-mobile` size can provide those styles. Let it be :smirk: